bitcoin-rpc-provider/src/lib.rs

```diff
@@ -227,6 +227,14 @@ impl Wallet for BitcoinCoreProvider {
             .map_err(rpc_err_to_manager_err)
     }
 
+    fn get_new_change_address(&self) -> Result<Address, ManagerError> {
```

Comment:
nit: since the code is the same maybe better to just call `get_new_address`?

Reply:
fixed